### ISSUE TITLE
roadmap - 0.0.6

### ISSUE THREAD

**Issue by JarbasAl** [OpenVoiceOS/ovos-core#249]
what is needed for release 0.0.6

- [x] https://github.com/OpenVoiceOS/ovos-core/pull/266
- [x] https://github.com/OpenVoiceOS/ovos-core/pull/245
- [x] https://github.com/OpenVoiceOS/ovos-core/pull/246
- [x] https://github.com/OpenVoiceOS/ovos-plugin-manager/pull/99
- [x] https://github.com/OpenVoiceOS/OVOS-workshop/pull/34
- [x] https://github.com/OpenVoiceOS/ovos-core/issues/199
- [x] https://github.com/OpenVoiceOS/ovos-ocp-audio-plugin/pull/27
- [x] https://github.com/OpenVoiceOS/ovos-ocp-audio-plugin/pull/34
- [x] ovos_utils stable release
- [x] LF stable release
- [x] OCP stable release
- [x] OCP plugins initial stable release and pypi automations
- [x] OPM stable release
- [x] OSM stable release
- [x] workshop stable release

Nice to have:
- [ ] https://github.com/OpenVoiceOS/ovos-core/pull/232
- [ ] bump version for https://github.com/Uberi/speech_recognition
- [ ] bump version for https://github.com/cmusphinx/pocketsphinx in https://github.com/OpenVoiceOS/ovos-ww-plugin-pocketsphinx
- [ ] https://github.com/OpenVoiceOS/ovos-core/pull/163
- [ ] https://github.com/OpenVoiceOS/skill-ovos-setup/pull/64
- [ ] https://github.com/OpenVoiceOS/ovos-backend-client/pull/11
- [ ] https://github.com/OpenVoiceOS/ovos-lingua-franca/pull/50
- [ ] https://github.com/OpenVoiceOS/ovos-lingua-franca/pull/51

current changelog: https://github.com/OpenVoiceOS/ovos-core/compare/V0.0.5...dev

**Implemented enhancements:**

- Add `skills-essential` optional dependencies [\#266](https://github.com/OpenVoiceOS/ovos-core/pull/266) ([NeonDaniel](https://github.com/NeonDaniel))
- sync lang resouces with mycroft-core [\#199](https://github.com/OpenVoiceOS/ovos-core/issues/199)
- Lng: Add cs-cz noise words [\#260](https://github.com/OpenVoiceOS/ovos-core/pull/260) ([NeonDaniel](https://github.com/NeonDaniel))
- Refactor/skill class from workshop [\#246](https://github.com/OpenVoiceOS/ovos-core/pull/246) ([JarbasAl](https://github.com/JarbasAl))
- add "last.voc" resource [\#248](https://github.com/OpenVoiceOS/ovos-core/pull/248) ([emphasize](https://github.com/emphasize))
- Add display setting for menu labels [\#244](https://github.com/OpenVoiceOS/ovos-core/pull/244) ([AIIX](https://github.com/AIIX))
- relax lingua\_franca version [\#242](https://github.com/OpenVoiceOS/ovos-core/pull/242) ([builderjer](https://github.com/builderjer))
- Reorg shutdown to prevent logged exceptions in skill shutdown [\#240](https://github.com/OpenVoiceOS/ovos-core/pull/240) ([NeonDaniel](https://github.com/NeonDaniel))
- Add config option to retry mic init on start \(default True\) [\#238](https://github.com/OpenVoiceOS/ovos-core/pull/238) ([NeonDaniel](https://github.com/NeonDaniel))
- page background color support for show image and show animated image [\#236](https://github.com/OpenVoiceOS/ovos-core/pull/236) ([AIIX](https://github.com/AIIX))
- add factory reset ui [\#231](https://github.com/OpenVoiceOS/ovos-core/pull/231) ([AIIX](https://github.com/AIIX))

**Fixed bugs:**

- Actually use the advertised audio device and sample rate with the audio test. [\#265](https://github.com/OpenVoiceOS/ovos-core/pull/265) ([gmsoft-tuxicoman](https://github.com/gmsoft-tuxicoman))
- Bump minimum plugin version for ocp fix [\#259](https://github.com/OpenVoiceOS/ovos-core/pull/259) ([NeonDaniel](https://github.com/NeonDaniel))
- Prevent logging errors when a blacklisted skill is handled as expected [\#257](https://github.com/OpenVoiceOS/ovos-core/pull/257) ([NeonDaniel](https://github.com/NeonDaniel))
- Ignore setuptools pip-audit failure [\#253](https://github.com/OpenVoiceOS/ovos-core/pull/253) ([NeonDaniel](https://github.com/NeonDaniel))
- Prevent exceptions if a user doesn't reply to an `ask_yesno` prompt [\#252](https://github.com/OpenVoiceOS/ovos-core/pull/252) ([NeonDaniel](https://github.com/NeonDaniel))
- Fix system url web view [\#251](https://github.com/OpenVoiceOS/ovos-core/pull/251) ([AIIX](https://github.com/AIIX))
- voice service has multiple errors, DeviceAPI and Hotword engine [\#216](https://github.com/OpenVoiceOS/ovos-core/issues/216)
- Update factory reset message [\#247](https://github.com/OpenVoiceOS/ovos-core/pull/247) ([NeonDaniel](https://github.com/NeonDaniel))
- port/fix/ intent name colision [\#233](https://github.com/OpenVoiceOS/ovos-core/issues/233)
- Handle multiple intents with the same name \(\#2921\) [\#235](https://github.com/OpenVoiceOS/ovos-core/pull/235) ([JarbasAl](https://github.com/JarbasAl))
- Fix display settings ui [\#237](https://github.com/OpenVoiceOS/ovos-core/pull/237) ([AIIX](https://github.com/AIIX))
- Fix show\_pages index api [\#234](https://github.com/OpenVoiceOS/ovos-core/pull/234) ([AIIX](https://github.com/AIIX))

**Comment by NeonDaniel** [comment]
@JarbasAl https://github.com/OpenVoiceOS/ovos-plugin-manager/pull/99 has review comments for you

**Comment by NeonDaniel** [comment]
Workshop waiting on LF stable release, but otherwise looks ready to release

**Comment by NeonDaniel** [comment]
OCP Plugins:
- [ ] https://github.com/OpenVoiceOS/ovos-ocp-files-plugin
- [ ] ovos-ocp-youtube-plugin
- [ ] ovos-ocp-deezer-plugin
- [ ] ovos-ocp-rss-plugin
- [ ] ovos-ocp-bandcamp-plugin
- [ ] ovos-ocp-m3u-plugin

**Comment by NeonDaniel** [comment]
Default skills:
- [ ] https://github.com/OpenVoiceOS/skill-ovos-setup/pull/69
- [ ] https://github.com/OpenVoiceOS/skill-ovos-homescreen/pull/66
- [ ] skill-fallback-unknown
- [ ] skill-stop
- [ ] skill-volume

**Comment by NeonDaniel** [comment]
Just need to bump dependency versions in ovos-core package now
